### PR TITLE
M0 branch

### PR DESCRIPTION
**Pull request by rohit1101**
Check if the initial setup is correct sir.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (digi0ps) [inline public/index.html:27]: Change title to `Go Retro`

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (digi0ps) [inline public/manifest.json:3]: Same here, update information to `Go Retro`

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
```
Comment (digi0ps) [inline src/App.test.js:2]: Delete this file. Not needed for now.

File: src/setupTests.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
```
Comment (digi0ps) [inline src/setupTests.js:3]: Remove this file too

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import logo from "./logo.svg";
```
Comment (digi0ps) [inline src/App.js:2]: Remove Logo as it is not being used.

File: src/logo.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
+    <g fill="#61DAFB">
```
Comment (digi0ps) [inline src/logo.svg:2]: Remove this file too

File: public/index.html (hunk 1 above)
Comment (rohit1101) [reply public/index.html:27]: Ok sir.

**Comment by rohit1101** [comment]
Making all the changes as one commit sir.

**Comment by digi0ps** [comment]
Merged manually, thanks! 🎉

**Pull request closed without merging**